moment.js

```diff
@@ -2033,7 +2058,46 @@
         }
     }
 
-    // date from iso format or fallback
+    // date from ietf format
+    function configFromIETF(config) {
+        var dateString, match, dateCheck, stDay;
+
+        dateString = config._i.
+        	replace(/(?:\([^\)]*\))/g, ' ').	// Replace comments with a single space
+        	replace(/[\l\n\r\t]/g, ' ').		// Replace 'fold' characters with a single space	
+        	replace(/\s{2,}/g, ' ').			// Replace multiple spaces with a single space
+        	replace(/(?:^\s|\s$)/g, '');		// Remove leading and training spaces	
+		match = dateString.match(ietfRegex);
+
+        if (match) {
+            getParsingFlags(config).ietf = true;
+            
+            dateCheck = new Date( match[2] + " GMT");
+            if (!dateCheck) {
+            	config._isValid = false;
```

Comment:
Don't set `_isValid`.  It's a cache variable set by the validity checks already in moment.